content/prerequisites/local-development/index.md

```diff
@@ -40,8 +40,7 @@ Default output format [None]: <leave blank>
 Clone the repository to your working directory:
 
 :::code{language=shell showLineNumbers=false showCopyAction=true}
-git clone https://github.com/aws-samples/cfn101-workshop
-:::
+git clone https://github.com/aws-samples/cfn101-workshop:::
```

Comment:
```suggestion
git clone https://github.com/aws-samples/cfn101-workshop
:::
```

Reply:
The suggested formatting breaks the generated code block.

Reply:
Running preview_build helped me fix this